src/base/_bootstrap.scss

```diff
@@ -0,0 +1,13 @@
+// Hack for Bootstrap 3.1.1, since it doesn't convert the "fadein" LESS method.
+// Pending fix release in https://github.com/twbs/bootstrap-sass/commit/df7be4f0412497e22f1d271d94d6e9835f61113e
+@function fadein($colour, $pct) {
```

Comment:
What version is needed for it to work? Nonetheless, this is something that can be addressed by another PR